python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -85,6 +86,8 @@ class ImageDifferenceConfig(pexConfig.Config):
     doWriteSubtractedExp = pexConfig.Field(dtype=bool, default=True, doc="Write difference exposure?")
     doWriteMatchedExp = pexConfig.Field(dtype=bool, default=False,
                                         doc="Write warped and PSF-matched template coadd exposure?")
+    doWriteWarpedTemplate = pexConfig.Field(dtype=bool, default=False,
```

Comment:
Actually, it's just added for debugging purposes, and just used for Zogy right now. But both tasks return a `warpedExposure` item in its return Struct.

But I do agree it would be nice to standardize the returned Struct.